docs/faq/app-locale.md

```diff
@@ -0,0 +1,138 @@
+Possible Values of `app.getLocale()`
```

Comment:
Our current style of docs requires the title to be like below:

```markdown
# Possible Values of `app.getLocale()`
```

https://github.com/electron/electron/blob/master/docs/styleguide.md#titles